ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -1086,6 +1112,22 @@ g_template_openshift_master:
     description: The application create loop has failed 4 or more times in the last hour
     priority: high
 
+  - name: "Route has returned non-200 HTTP code multiple times in the last hour on {HOST.NAME}"
+    expression: "{Template Openshift Master:openshift.master.app.create.route_http_failed.sum(1h)}>3"
+    url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_create_app.asciidoc"
+    dependencies:
```

Comment:
Is the dependency needed here?

Reply:
I copied the dependency from the app-create check, because if the master API goes offline, that means app creation will fail. And without an app to route the http traffic to, all curl requests are going to fail.